portfolio/src/main/webapp/style.css

```diff
@@ -4,6 +4,60 @@
   width: 650px;
 }
 
-#greeting-container {
-  margin-top: 20px;
+html{
+    height: 100%;
+    width: 100%;
 }
+ 
+body{
+    height: 100%;
+    margin: 0;
+    background: linear-gradient(45deg,
+      rgba(20, 18, 19, 0.9) 30%,
+      rgba(0, 94, 105, 0.9) 100%),
+    url(Images/github_pattern.svg);
+    background-repeat: no-repeat;
+    background-attachment: fixed;
+}
+ 
+#content {
+    margin-left: auto;
+    margin-right: auto;
+    width: 650px;
+}
+ 
+#title{
+    text-align: center;
+}
+ 
+h1{
+    color: white;
+}
+ 
+#text{
+    background: rgba(255,255,255,0.7); 
+    display: inline-block;
+}
+
+#image-container{
+    width: 20em;
+}
+ 
+#selfie{
+    object-fit: contain;
+    max-width: 100%;
+}
+
+#clickBtn{
+    
+}
+
+#language-container {
+    margin-top: 20px;
+}
+
+.end{
+    margin-bottom: 15px;
+    display: inline-block;
+    background-color: rgba(255,255,255,0.7);
+}
```

Comment:
Is there a reason why these have to be two different things? It seems like they're both attached to the same div.

Reply:
Ack!